--- a/upnpdescstrings.h	2010-11-12 07:48:13.000000000 +0800
+++ b/upnpdescstrings.h	2013-08-14 00:45:03.610899294 +0800
@@ -44,8 +44,8 @@
  #endif
 #else
  #define ROOTDEV_MANUFACTURERURL	OS_URL
- #define ROOTDEV_MANUFACTURER		"Justin Maggard"
- #define ROOTDEV_MODELNAME		"Windows Media Connect compatible (MiniDLNA)"
+ #define ROOTDEV_MANUFACTURER		"TP-LINK"
+ #define ROOTDEV_MODELNAME		"Windows Media Connect Compatible (MiniDLNA)"
  #define ROOTDEV_MODELDESCRIPTION	"MiniDLNA on " OS_NAME
  #define ROOTDEV_MODELURL		OS_URL
 #endif
